src/lib/collab/schema.js

```diff
@@ -1,26 +1,49 @@
 import { Schema } from 'prosemirror-model'
 
-// taken from a debug of @atlakit/editor/editor-core/create-editor/create-editor
-// L139 (new Schema({nodes ,marks}))
 // static because the @atlaskit code base requires a real navigator
 // TODO: either find and exclude plugins requiring interaction
 //       or running a JSDOM faking a navigator
+// @quentin: I don't understand yet why we need to do that, so if someone
+// knows, then update the comment ;)
 
-const nodes = [
+// To get the schema, we need to log the values from node_modules/@atlaskit/editor/editor-core/create-editor/create-schema
+// itself just before the call on new Schema({nodes, marks}) and then
+// console.log({node})
+// console.log({marks})
+// to be able to copy paste the value in this file, click right on the log -> store as global value
+// then copy(temp1) and then past it in the method written below.
+
+// const objectToArray = obj => {
+// const nodesArray = []
+//  Object.keys(obj).map(key => {
+//    nodesArray.push([key, obj[key]])
+//  })
+//  return nodesArray
+// }
+//
+// in order to convert it in Array.
+// It seems that Atlaskit is using Schema containing an Object but prosemissor an Array
+// Don't know yet why and if we can't reuse a code from atlaskit to do that transformation
+// since they have to do it somewhere I think
+
+//https://github.com/cozy/cozy-notes/pull/2/commits/202e529cdb1e71996c2cab43057984c9f885b61a
+
+export const nodes = [
   [
     'doc',
     {
       content: '(block)+',
-      marks: 'link'
+      marks: 'link unsupportedMark unsupportedNodeAttribute'
     }
   ],
   [
     'paragraph',
     {
+      selectable: false,
```

Comment:
Why do we need to put selectable: false ?

Reply:
I don't know... yet. To be honest, I don't know why we need to dump this internal schema from atlaskit yet.